sameer.json

```
@@ -0,0 +1,20 @@
+{
+     "description": "Sam Rajput's Portfolio ",
+     "repo": "https://github.com/MrErrOr95/https://MrErrOr95.github.io",
+     "owner": {
+         "username": "MrErrOr95",
+         "email": "jogi.95600@gmail.com",
+         "twitter": "unique_error"
+         "instagram": "unique__error"
+         "telegram": "UniQueErrOr"
+         "threads": "unique__error"
+         "facebook": "UniQue.ErrOr"
+         "facebook": "Mr.Sa1M"
+         "youtube": "uniqueerror"
+         "snapchat": "unique.error"
+         "linkedin": "joginder-singh-rajput"
+     },
+     "record": {
+         "CNAME": "MrErrOr95.github.io"
```

could you add an about me or something as the website doesnt have anything related to development
```suggestion
         "CNAME": "mrerror95.github.io"
```